Formula/lasi.rb

```diff
@@ -27,7 +28,21 @@ def install
     # None is valid, but lasi's CMakeFiles doesn't think so for some reason
     args = std_cmake_args - %w[-DCMAKE_BUILD_TYPE=None]
```

Comment:
`std_cmake_args` contains `-DCMAKE_BUILD_TYPE=Release` nowadays, not `-DCMAKE_BUILD_TYPE=None`, so this all thing can be removed.